app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -85,7 +85,17 @@ module(HELP_CONTENTS, [])
         the configuration based on the newest server group in the cluster.</p>
         <p>If you want to start from scratch, select "None".</p>
         <p>You can always edit the cluster configuration after you've created it.</p>`,
-
+    'pipeline.config.expectedArtifact.fieldName': `
+        <p>The Artifact field name.</p>`,
+    'pipeline.config.expectedArtifact.fieldType': `
+        <p>The Artifact field type. This must be one of 'MUST_MATCH' or 'FIND_IF_MISSING'.</p>
+        <p><strong>MUST_MATCH</strong> means the incoming Artifact's field value must match exactly with the expected value.</p>
+        <p><strong>FIND_IF_MISSING</strong> means the field is optional and a 'Missing Policy' will fire if the value is not present.</p>`,
+    'pipeline.config.expectedArtifact.missingPolicy': `
+        <p>The policy if the field type is 'FIND_IF_MISSING' and the field value is not present.</p>
```

Comment:
This can be an incremental improvement for another PR, but it would be better if people didn't have to look at enums. Take a look at https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/pipeline/config/stages/stageConstants.ts.

Reply:
I'll take this as improvements on artifact UI v2 -- I'll create a tracking bug for these.